Evernote.DataContext/Unit/UnitOfWork.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Evernote.DataContext.Generic;
+using Evernote.DataContext.Repositories;
+
+namespace Evernote.DataContext.Unit {
+    public class UnitOfWork : IUnitOfWork {
+
+
+#pragma warning disable IDE0025
+
+        public ITagRepository TagRepository => new TagRepository(_dbContext);
```

Comment:
this is not a DI pattern. Move the initialization to the constructor. Or better yet, use Lazy, This will improve performance by creating only the necessary repository instance, for example:

```csharp
private readonly Lazy<ITagRepository> _tagRepository;

public UnitOfWork(AppDataContext context)
{
    _context = context
    _tagRepository = new Lazy<ITagRepository>(() => new TagRepository(_context))
}

ITagRepository TagRepository => _tagRepository.Value
```